CHANGES.rst

```diff
@@ -95,6 +95,9 @@ There's a frood who really knows where his towel is.
 - Changes image link behavior in view.pt for mobile devices
   (closes `#62`_). [marcosfromero]
 
+- Replace Galleria with Cycle2 (closes `#116`_).
```

Comment:
sorry about that.. getting difficult to change the way I write change logs in the last years..